x-pack/legacy/plugins/ml/public/application/data_frame_analytics/common/analytics.ts

```diff
@@ -74,13 +75,26 @@ export interface RegressionEvaluateResponse {
   };
 }
 
+export interface ClassificationEvaluateResponse {
+  classification: {
+    multiclass_confusion_matrix: {
+      confusion_matrix: Array<{
+        actual_class: string;
+        actual_class_doc_count: number;
+        predicted_classes: Array<{ predicted_class: string; count: number }>;
```

Comment:
Suggest to create an interface `PredictedClass` and use it here like `predicted_classes: PredictedClass[];`